packages/odyssey-react-mui/src/labs/DataView/DataView.test.tsx

```diff
@@ -21,8 +21,8 @@ import { DataView } from "./index";
 import {
   data,
   columns,
-} from "@okta/odyssey-storybook/src/components/odyssey-labs/DataView/personData";
-import { filterData } from "@okta/odyssey-storybook/src/components/odyssey-labs/DataView/dataFunctions";
+} from "../../../../odyssey-storybook/src/components/odyssey-labs/DataView/personData";
+import { filterData } from "../../../../odyssey-storybook/src/components/odyssey-labs/DataView/dataFunctions";
```

Comment:
These files shouldn't be imported from `@okta/odyssey-storybook` as that's not a dependency of `@okta/odyssey-react-mui`. In fact, that creates a cyclical import. It was causing issues while I was running tests, so I had to fix this.

Reply:
Should these things actually go in some separate package to be imported by each? Seems bad practice to use relative paths outside of this package's dir but acceptable in a pinch.

Reply:
For sure, there should eventually be a real fix. I'll get them to make a ticket.